fix: Store UI fixes

illmavir1 opened the pull request:
closes #88

ericselin reviewed (changes requested):
Good stuff, and mostly CSS edits, which is nice 🙃 First immediate comment is regarding the menu. It still needs a little bit of polishing. Below some observations.

The margins in the menu are inconsistent:
![image](https://user-images.githubusercontent.com/27487150/102472983-2f9bbc80-4057-11eb-91de-deb0bfd4da1c.png)

Also on mobile. I also personally don't think the line with margin on one end but not the other is weird. But don't change it if you don't think so.
![image](https://user-images.githubusercontent.com/27487150/102473258-8a351880-4057-11eb-9d1c-88d2d03cda2e.png)

The padding on opened menu items I do not really get. But if that's the way it's designed, then it's great. 😅

More reviews to follow!

illmavir1 commented:
I agree menu paddings were inconsistent. Those were inconsistent also on design. 
I fixed menu list items to have same top and bottom paddings. And removed also that extra bottom padding from opened sub items.

ericselin commented on layouts/_default/baseof.header-critical.css:
```diff
@@ -6,61 +6,103 @@ input[type="checkbox"][id|="open"] {
 .menu {
   display: flex;
   align-items: center;
+  height: var(--menu-height);
```
Why does the height need to be adjusted here?

illmavir1 replied:
Because height is bigger on desktop and smaller on mobile and tablet

ericselin commented on layouts/_default/baseof.header-critical.css:
```diff
@@ -6,61 +6,103 @@ input[type="checkbox"][id|="open"] {
 
-.menu input:focus + label.menu__hamburger {
+.menu input:focus + label.menu__hamburger,
+.menu
+  input:focus
+  + label
+  + .overlay
+  > .overlay__inner
+  > .overlay__header
+  > label {
```
There has to be an easier selector than this. Or?

illmavir1 replied:
I can take overlay__inner out then it would look a bit nicer and still works fine on current menu.
.menu input:focus + label + .overlay .overlay__header > label

ericselin reviewed (commented):
This PR is way too big. 30 changed files and hundreds of lines changed in one PR (one commit even). It makes this extremely difficult to review, since it's impossible to get an overview of what's been done and what the reasoning behind different choices has been.

👍 for working on this independently and figuring everything out by yourself, though! 🙃

We basically have two options: either I review this properly or then we just merge this as-is. What do you think, is this as good as it can be? I.e. is it ready to be pushed live as-is?

I already made some scattered comments here, let's see what we do with them and if we implement any changes based on them.

illmavir1 commented on layouts/_default/baseof.footer.css:
```diff
@@ -58,6 +67,24 @@ footer > p {
 
 footer label {
   fill: white;
+  position: absolute;
+  left: 0;
+  right: 0;
+}
+
+footer label.icon {
+  width: 100%;
+}
+
+footer label.icon svg {
+  width: auto;
+  position: absolute;
+  right: 0;
+}
+
+footer input:focus + label.icon {
+  outline: var(--focus-outline);
+  outline-offset: 2px;
```
Copied footer items from us shop to demo

illmavir1 commented on layouts/_default/baseof.footer.css:
```diff
@@ -49,7 +49,16 @@ footer ul ul {
   text-decoration: none;
-  flex: 1 1 auto;
+  font-size: 0.875rem;
+  z-index: 1;
```
z-index is here because some footer headers had links on us shop. Without z-index link is unclickable on mobile because toggle area was expanded to whole row.

illmavir1 commented on layouts/_default/baseof.header-critical.css:
```diff
@@ -6,61 +6,103 @@ input[type="checkbox"][id|="open"] {
+  padding: 0.5rem 0;
+}
+
+.announcement .announcement--content {
+  padding: var(--block-padding);
+  margin: auto;
+  max-width: var(--block-width);
```
That element is to keep announcement content width same as other content on page. 
On proto there is .content-container element (https://www.reimadesign.com/overlay-menu)

illmavir1 commented on layouts/_default/baseof.header-critical.css:
```diff
@@ -6,61 +6,103 @@ input[type="checkbox"][id|="open"] {
+
+.announcement .announcement--content {
+  padding: var(--block-padding);
+  margin: auto;
+  max-width: var(--block-width);
 }
 
 .announcement a {
-  color: inherit;
+  color: white;
```
Nothing wrong. I'm not sure on which point and why I changed this but I can revert it.

illmavir1 commented:
This is huge PR but I think git is also making some files looks uglier that changes really are.
Most of changes are still just some formatting and taking use of CSS variables. So I'm pretty sure it's good as is. 

You know the current version best and might have good points why something should be as it was. So I wish you would have time to check this thru.
I also commented back to your previous comments. So if you like I can do those couple small changes.

ericselin commented:
:tada: This PR is included in version 2.5.1 :tada:

The release is available on [GitHub release](https://github.com/reima-ecom/reima-theme/releases/tag/v2.5.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
